.gitignore

```diff
@@ -2,3 +2,4 @@
 .idea
 **/*.rs.bk
 Cargo.lock
+.vscode/
```

Comment:
this should be removed (we won't start adding special case for every editor)

Reply:
done